README_fr-FR.md

```diff
@@ -0,0 +1,443 @@
+For les fichiers qui ne changent pas d'un commit à l'autre, git stock just un lien vers la version précédente identique qui est déjà stockée.
+
+L'image ci-dessous montre comment git stock les fichiers à travers le temps, dans quelle "Version" est un commit.
+
+
+![](https://i.stack.imgur.com/AQ5TG.png)
+
+## Pourquoi les messages de commit sont importants ?
+
+- Pour raccourcir et faciliter les code-review
+- Pour aider dans la compréhension d'un changement.
+- Pour expliquer les "pourquoi", qui ne sauraient être décris en code
+- Pour aider les futurs personnes qui vont devoir maintenir le code à comprendre les différents changements, et l'évolution du code.
+
+Pour maximiser ces résultats, nous pouvons utiliser certaines bonnes pratiques et des standards, décrits dans la prochaine section.
+
+## Bons Usages
+
+Il y a quelques bons usages recueillit par l'experience, des articles sur internet, et d'autres guides. Si vous en avez d'autres (ou n'êtes pas d'accord avec certains), soyez libres d'ouvrir une demande de Pull et de contribuer à l'amélioration du guide.
+
+### Utilisez l'impératif
+
+```
+# Bon
+Use InventoryBackendPool to retrieve inventory backend
+```
+
+```
+# Mauvais 
+Used InventoryBackendPool to retrieve inventory backend
+```
+
+_Pourquoi l'impératif ?__
+
+Un message de commit décrit ce que le changement **fait**, ces effets, pas ce qui à été fait.
+
+[Cet excelent article de Chris Beams](https://chris.beams.io/posts/git-commit/)
+Nous donne une phrase simple qui peut être utilisée pour mieux écrire les messages de commit à l'impératif.
+
+```
+If applied, this commit will <commit message>
+```
+
+Examples:
+
+```
+# Bon
+If applied, this commit will use InventoryBackendPool to retrieve inventory backend
+```
+
+```
+# Mauvais
+If applied, this commit will used InventoryBackendPool to retrieve inventory backend
+```
+
+### Capitalize the first letter
+
+```
+# Bon
+Add `use` method to Credit model
+```
+
+```
+# Mauvais
+add `use` method to Credit model
+```
+
+The reason that the first letter should be capitalized is to follow the grammar rule of using capital letters at the beginning of sentences.
+
+The use of this practice may vary from person to person, team to team, or even from language to language.
+Capitalized or not, an important point is to stick to a single standard and follow it.
+
+### Try to communicate what the change does without having to look at the source code
+
+```
+# Bon
+Add `use` method to Credit model
+
+```
+
+```
+# Mauvais
+Add `use` method
+```
+
+```
+# Bon
+Increase left padding between textbox and layout frame
+```
+
+```
+# Mauvais
+Adjust css
+```
+
+It is useful in many scenarios (e.g. multiple commits, several changes and refactors) to help reviewers understand what the committer was thinking.
+
+### Use the message body to explain "why", "for what", "how" and additional details
```

Comment:
`Use the message body to explain "why", "for what", "how" and additional details` > `Utilisez le corps du message pour expliquer "pourquoi", "pour quoi", "comment" et des détails supplémentaires`